src/Microsoft.IdentityModel.KeyVaultExtensions/KeyVaultSecurityKey.cs

```diff
@@ -90,7 +90,7 @@ public override int KeySize
                 if (!_keySize.HasValue)
                     Initialize();
 
-                return _keySize.Value;
+                return _keySize!.Value;
```

Comment:
Once initialize is used, keySize would not be null? unless getKeyAsync throws

Reply:
That is my understanding as well.